python/run-tests.py

```diff
@@ -71,7 +71,18 @@ def get_valid_filename(s):
     raise RuntimeError("Cannot find assembly build directory, please build Spark first.")
 
 
-def run_individual_python_test(target_dir, test_name, pyspark_python):
+def run_individual_python_test(target_dir, test_name, pyspark_python, keep_test_output):
+    """
+    Runs an individual test. This function is called by the multi-process runner of all tests.
+
+    Parameters
+    ----------
+    target_dir Destination for the Hive and log directory.
+    test_name  Test name.
+    pyspark_python Python version used to run the test.
+    keep_test_output Flag indicating if the test output should be retained after successful execution.
```

Comment:
```suggestion
    target_dir
        Destination for the Hive and log directory.
    test_name 
        Test name.
    pyspark_python
        Python version used to run the test.
    keep_test_output
        Flag indicating if the test output should be retained after successful execution.
```

Per https://numpydoc.readthedocs.io/en/latest/format.html